Makefile

```diff
@@ -197,11 +203,16 @@ endif
 
 
 ifeq ($(detected_OS),Darwin)
-  ifeq ("$(shell sysctl -nq hw.optional.arm64)","1")
-    # Building on M1 is still not supported, so in the meantime we crosscompile to amd64
-    DOTHERSIDE_CMAKE_PARAMS += -DCMAKE_OSX_ARCHITECTURES=x86_64
-	NIM_PARAMS += --cpu:amd64 --os:MacOSX --passL:"-arch x86_64" --passC:"-arch x86_64"
+ ifeq ("$(shell sysctl -nq hw.optional.arm64)","1")
+  ifeq ($(QT_ARCH),arm64)
+   STATUSGO_MAKE_PARAMS += FORCE_ARCH=arm64
```

Comment:
There is no need for this particular case. The `FORCE_ARCH` is used only for forcing `x64_86` on Darwin `arm64` platforms. Now that you have added support for building on apple silicon, the `FORCE_ARCH` should be removed from status-go. That means the following reverted PR should be included with your PR https://github.com/status-im/status-go/pull/3131

Reply:
Good point, thanks! I'll update status-keycard-go as well. I've aligned the make config options to support `FORCE_ARCH` as well, but I see it was a mistake.

Reply:
@stefandunca I've updated the Makefile and drafted the PRs for status-go and status-keycard-go (https://github.com/status-im/status-go/pull/3288 , https://github.com/status-im/status-keycard-go/pull/5)

Please have a look.